content/projects/github-api-consume/_index.md

```diff
@@ -48,30 +48,37 @@ While for private repos a token will be required, the function should be able to
 
 Please be sure to follow the standard naming conventions for your language.
 
-Make sure the output matches the following example structure:
-```
+Make sure the output matches the following when called with the arguments below:
+
+```bash
 # input
 get_pull_requests("Umuzi-org", "ACN-syllabus", "2022-03-01", "2022-03-10")
 
 # output
 [
-  {"id":876359209, "user":"FaithMo", "title":"added data sci and eng info", "state":"open", "created_at":"2022-03-10"},
+  {"id":"876359209", "user":"FaithMo", "title":"added data sci and eng info", "state":"open", "created_at":"2022-03-10"},
   {"id":"874927260", "user":"ry-oc", "title":"update sololearn python and all contentlinks etc", "state":"closed", "created_at": "2022-03-09"},
-  {"id":872630389, "user":"Andy-Nkumane", "title":"added clarity on python error raising", "state":"open", "created_at":"2022-03-07"},
+  {"id":"872630389", "user":"Andy-Nkumane", "title":"added clarity on python error raising", "state":"open", "created_at":"2022-03-07"},
   {"id":"872484561", "user":"Kate-bit-dev", "title":"Update _index.md", "state":"closed", "created_at":"2022-03-06"},
   {"id":"872482562", "user":"Kate-bit-dev", "title":"Update _index.md", "state":"open", "created_at":"2022-03-06"},
   {"id":"872481470", "user":"Kate-bit-dev", "title":"Update _index.md", "state":"closed", "created_at":"2022-03-06"},
   {"id":"872480774", "user":"Kate-bit-dev", "title":"Update _index.md", "state":"closed", "created_at":"2022-03-06"},
-  {"id":"872480210", "user":"Kate-bit-dev", "title":"Update _index.md", "state":"closed", "created_at":"2022-03-06"}
+  {"id":"872480210", "user":"Kate-bit-dev", "title":"Update _index.md", "state":"closed", "created_at":"2022-03-06"},
+  ...
 ]
 ```
 
+**Note:** Some github APIs have a limit of the number of resources they return at a time. You might be expected to implement pagination to get all resources from the APIs before filtering.
```

Comment:
If there is a 10% chance that the learner will misinterpret the sentence, then 10% of learners will misinterpret the sentence...

Maybe be more explicit.  Eg:

```
Pagination is ... (say something about why it is a useful thing to know about). 

Please implement pagination.  Your get_pull_requests function should return a list/array of ALL the matching pull requests, not just the first page.
```